SPECS/python-trove-classifiers/python-trove-classifiers.spec

```diff
@@ -1,32 +1,36 @@
-# Disable tests as it requires new package python-exceptiongroup
-%global with_check 0
 Vendor:         Microsoft Corporation
 Distribution:   Azure Linux
 
 %undefine distro_module_ldflags
 
-%global pypi_name trove-classifiers
 Name:           python-trove-classifiers
-Version:        2024.2.23
-Release:        2%{?dist}
+Version:        2024.3.3
+Release:        1%{?dist}
 Summary:        Canonical source for classifiers on PyPI (pypi.org)
 
 License:        Apache-2.0
 URL:            https://github.com/pypa/trove-classifiers
-Source:         %{pypi_source trove-classifiers}
+Source:         https://github.com/pypa/trove-classifiers/archive/refs/tags/2024.3.3.tar.gz#/trove-classifiers-%{version}.tar.gz
 
-# [IMPORTANT] Update the patch for every new version
-Patch:          001_remove_claver_dependency.patch
+
+# Drop dependency on calver and use PEP 621 declarative metadata.
+# This patch is rebased version of upstream PR:
+# https://github.com/pypa/trove-classifiers/pull/126/commits/809156bb35852bcaa1c753e0165f1814f2bcedf6
+Patch: Move-to-PEP-621-declarative-metadata.patch
 
 BuildArch:      noarch
 BuildRequires:  python3-devel
+%if 0%{?with_check}
+BuildRequires:  python3-pytest
+#BuildRequires:  python3-iniconfig
+%endif
+
 BuildRequires:  pyproject-rpm-macros
 BuildRequires:  python3-pip
 BuildRequires:  python3-setuptools
 BuildRequires:  python3-six
 BuildRequires:  python3-wheel
 BuildRequires:  python3-packaging
-BuildRequires:  python3-toml
```

Comment:
We should also remove this from `python-tox.spec`.

Reply:
python-toml is still in SPECS-EXTENDED. I only removed it from SPECS. Can remove from both if desired, though I think I'd make that a separate PR.

Reply:
Wow, I didn't notice it still lingering in SPECS-EXTENDED. OK, it looks like the build shouldn't break then, I'm fine leaving the clean-up for later.